src/date-navigator/date-navigator.tsx

```diff
@@ -0,0 +1,115 @@
+import dayjs from "dayjs";
+import {
+    ArrowLeft,
+    ArrowRight,
+} from "../shared/internal-calendar/calendar-manager.style";
+import { CalendarHelper, DateHelper } from "../util";
+import {
+    Container,
+    HeaderArrowButton,
+    StyledDateText,
+    StyledDayText,
+    Wrapper,
+} from "./date-navigator.style";
+import { DateNavigatorProps } from "./types";
+
+export const DateNavigator = ({
+    selectedDate,
+    ...optionalProps
```

Comment:
can destructure props that are being accessed for processing/logic like `minDate` and `onLeftArrowClick`, the remaining props can be passed to `<Container {...otherProps} />`